lib/ansible/modules/network/nxos/nxos_vrf.py

```diff
@@ -242,21 +242,21 @@ def map_obj_to_commands(updates, module):
         elif state == 'present':
             if not obj_in_have:
                 commands.append('vrf context {0}'.format(name))
-                if rd and rd != '':
-                    commands.append('rd {0}'.format(rd))
-                if description:
-                    commands.append('description {0}'.format(description))
-                if vni and vni != '':
-                    commands.append('vni {0}'.format(vni))
+                for item in args:
+                    candidate = w.get(item)
+                    if candidate:
+                        cmd = item + ' ' + str(candidate)
+                        commands.append(cmd)
                 if admin_state == 'up':
                     commands.append('no shutdown')
                 elif admin_state == 'down':
                     commands.append('shutdown')
 
                 if commands:
```

Comment:
Probably not related to this diff but what is the use of this if command when command is already declared so it seems like a  if True type of check.

Reply:
Good point. I will remove this check.